test/Tizen.NUI.ThemeManager.Samples/TizenThemeChangeSample/TizenThemeChangeSample.cs

```diff
@@ -0,0 +1,299 @@
+        };
+        closeButton.Clicked += (s, e) => {
+            Exit();
+        };
+
+        var mainPage = new ContentPage()
+        {
+            ThemeChangeSensitive = true,
+            AppBar = new AppBar()
+            {
+                AutoNavigationContent = false,
+                Title = "NUI theme sample",
+                Actions = new View[] { closeButton },
+            },
+            Content = new ScrollableBase()
+            {
+                Layout = new LinearLayout()
+                {
+                    LinearOrientation = LinearLayout.Orientation.Vertical,
+                    CellPadding = new Size2D(20, 20)
+                },
+                WidthResizePolicy = ResizePolicyType.FillToParent,
+                HeightResizePolicy = ResizePolicyType.FillToParent,
+                Padding = new Extents(30, 30, 20, 20)
+            }
+        };
+
+        var title = $"Click to change theme to {(ThemeManager.PlatformThemeId == ThemeManager.DefaultLightThemeName ? "Dark" : "Light")}";
+
+        mainPage.Content.Add(CreateClickableItem("Theme change", title, delegate(View view) {
+            TextLabel text = view.Children[1] as TextLabel;
+
+            if (ThemeManager.PlatformThemeId == ThemeManager.DefaultDarkThemeName)
+            {
+                ThemeManager.ApplyPlatformTheme(ThemeManager.DefaultLightThemeName);
+                text.Text = "Click to change theme to Dark";
+            }
+            else
+            {
+                ThemeManager.ApplyPlatformTheme(ThemeManager.DefaultDarkThemeName);
+                text.Text = "Click to change theme to Light";
+            }
+        }));
+
+        mainPage.Content.Add(CreateItem("Switch", CreateSwitchExample()));
+        
+        mainPage.Content.Add(CreateItem("RadioButton", CreateRadioButtonExample()));
+        
+        mainPage.Content.Add(CreateClickableItem("AlertDialog", "Click to post alert", delegate(View view) {
+            Button button;
+            DialogPage.ShowAlertDialog("Title", "Message", button = new Button() { Text = "Close" });
+            button.Clicked += (s, e) => {
+                NUIApplication.GetDefaultWindow().GetDefaultNavigator().Pop();
+            };
+        }));
+
+        mainPage.Content.Add(CreateItem("CheckBox", CreateCheckBoxExample()));
+
+        mainPage.Content.Add(CreateClickableItem("Exit", "Click to exit application", delegate(View view) {
+            Exit();
+        }));
+
+        NUIApplication.GetDefaultWindow().GetDefaultNavigator().Push(mainPage);
+    }
+
+    private View CreateItem(string title, View content)
+    {
+        var item = new View()
+        {
+            WidthResizePolicy = ResizePolicyType.FillToParent,
+            HeightResizePolicy = ResizePolicyType.FitToChildren,
+            Layout = new LinearLayout()
+            {
+                LinearOrientation = LinearLayout.Orientation.Vertical,
+            },
+            BackgroundColor = new Color("#88888860"),
```

Comment:
This color value is widely used a lot. I think.
How about making this string to const value?

Reply:
That's a good point. I updated the patch as you suggested. Thank you!